Nos/Views/StoriesView.swift

```diff
@@ -7,84 +7,84 @@
 
 import SwiftUI
 
+struct StoriesDestination: Hashable {
+    var author: Author?
+}
+
 struct StoriesView: View {
     
     @ObservedObject var user: Author
+
+    @EnvironmentObject var router: Router
     
     @FetchRequest private var authors: FetchedResults<Author>
     
-    @State private var currentAuthorIndex: Int = 0
-    @State private var scrollViewProxy: ScrollViewProxy?
+    @State private var selectedAuthor: Author
     @Binding private var cutoffDate: Date
     
-    init(user: Author, cutoffDate: Binding<Date>) {
+    init(user: Author, cutoffDate: Binding<Date>, selectedAuthor: Author? = nil) {
         self.user = user
         self._cutoffDate = cutoffDate
         _authors = FetchRequest(fetchRequest: user.followedWithNewNotes(since: cutoffDate.wrappedValue))
+        _selectedAuthor = .init(initialValue: selectedAuthor ?? Author())
     }
     
     var body: some View {
         VStack {
-            ScrollView(.horizontal) {
-                ScrollViewReader { proxy in
-                    HStack {
-                        ForEach(authors) { author in
-                            let index = authors.firstIndex(of: author)
-                            Button { 
-                                if let index {
-                                    currentAuthorIndex = index
-                                }
-                            } label: { 
-                                let avatar = AvatarView(imageUrl: author.profilePhotoURL, size: 54)
-                                    .padding(.vertical, 8)
-                                    .padding(.horizontal, 4)
-                                
-                                if let index, index == currentAuthorIndex {
-                                    avatar
-                                        .background(
-                                            Circle()
-                                                .stroke(LinearGradient.diagonalAccent, lineWidth: 3)
-                                                .frame(width: 58, height: 58)
-                                        )
-                                } else {
-                                    avatar
-                                }
-                            }
-                            .id(index)
-                            .onAppear {
-                                scrollViewProxy = proxy
-                            }
-                        }
-                    }
+            TabView(selection: $selectedAuthor) {
```

Comment:
Replaces a ScrollView to navigate between authors to TabView because it is semantically more precise and probably more performant